libraries/load-themed-styles/src/index.ts

```diff
@@ -10,7 +10,9 @@
 
 // Declaring a global here in case that the execution environment is Node.js (without importing the
 // entire node.js d.ts for now)
+/// <reference lib="dom" />
```

Comment:
Is this needed anymore now that you extend from the web rig?

Reply:
No